thaomonster/ACNH-Collections
# Feature/acessibilty media queries

### What does this PR do? 
>Clearly and concisely describe the feature. 

This PR adds media queries to different screen sizes. Is responsive and has 100% accessibility.

### Is this a feature or Fix/Refactor?  
Feature

### Problems Encountered & Resulting Solutions  
>State any problems encountered 
>Describe your code changes in detail for reviewers.

None.

### Where should the reviewer start?
>What file(s) and line(s) are where changes are made?
`src/styles/_mediaQueries.scss` lines: 1-88;

### How should this be tested?
>What steps need to be taken to confirm functionality? 
1. `git fetch` branch
2. run `npm start`
3. Check out different screensizes
4. Run lighthouse

### Steps To Take
>Any questions and possible steps moving forward.

None.

### This PR is related to which issues:
>Tag issue here: `Closes #<issue_number>`
Closes #25
